applications/RANSApplication/python_scripts/coupled_rans_solver.py

```diff
@@ -0,0 +1,303 @@
+# Importing the Kratos Library
+import KratosMultiphysics as Kratos
+from KratosMultiphysics import IsDistributedRun
+from KratosMultiphysics.kratos_utilities import CheckIfApplicationsAvailable
+from KratosMultiphysics.python_solver import PythonSolver
+
+# Import applications
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+
+# Import application specific modules
+from KratosMultiphysics.RANSApplication.formulations import Factory as FomulationFactory
+from KratosMultiphysics.RANSApplication import RansVariableUtilities
+from KratosMultiphysics.FluidDynamicsApplication.check_and_prepare_model_process_fluid import CheckAndPrepareModelProcess
+
+# case specific imports
+if (IsDistributedRun() and CheckIfApplicationsAvailable("TrilinosApplication")):
+    import KratosMultiphysics.TrilinosApplication as KratosTrilinos     # MPI solvers
+    from KratosMultiphysics.mpi.distributed_import_model_part_utility import DistributedImportModelPartUtility
+elif (IsDistributedRun()):
+    raise Exception("Distributed run requires TrilinosApplication")
+
+class CoupledRANSSolver(PythonSolver):
+    @classmethod
+    def GetDefaultSettings(cls):
+        ##settings string in json format
+        default_settings = Kratos.Parameters("""
+        {
+            "solver_type": "CoupledRANS",
+            "model_part_name": "FluidModelPart",
+            "domain_size": -1,
+            "model_import_settings": {
+                "input_type": "mdpa",
+                "input_filename": "unknown_name",
+                "reorder": false
+            },
+            "material_import_settings": {
+                "materials_filename": ""
+            },
+            "consider_periodic_conditions": false,
+            "formulation_settings": {},
+            "wall_function_settings": {},
+            "echo_level": 0,
+            "volume_model_part_name": "volume_model_part",
+            "skin_parts"   : [""],
+            "no_skin_parts": [""],
+            "assign_neighbour_elements_to_conditions": true,
+            "move_mesh": false,
+            "time_scheme_settings":{
+                "scheme_type": "steady"
+            },
+            "time_stepping": {
+                "automatic_time_step" : false,
+                "CFL_number"          : 1,
+                "minimum_delta_time"  : 1e-4,
+                "maximum_delta_time"  : 0.01,
+                "time_step"           : 0.0
+            },
+            "constants": {}
+        }""")
+
+        default_settings.AddMissingParameters(
+            super(CoupledRANSSolver, cls).GetDefaultSettings())
```

Comment:
```suggestion
            super().GetDefaultSettings())
```

just to say that this is no longer needed with python3